modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,308 @@
+
+[NOTE]
+====
+For more information about the configuration resources that you must configure in the hub cluster, see _Cluster configuration resources for deploying a preinstalled host_.
+====
+
+.Prerequisites
+
+* You preinstalled a host with {sno} using an image-based installation.
+* You logged in as a user with `cluster-admin` privileges.
+* You deployed a {rh-rhacm-first} hub cluster or you deployed the multicluster engine for Kubernetes operator (MCE).
+* You installed the IBI Operator on the hub cluster.
+* You created a pull secret to authenticate pull requests. See the _Additional resources_ section for more information about using image pull secrets.
+
+.Procedure
+
+. Create the `ibi-ns` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace ibi-ns
+----
+
+. Create the `Secret` resource for your image registry:
+
+.. Create a YAML file that defines the `Secret` resource for your image registry:
++
+.Example `secret-image-registry.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ibi-image-pull-secret
+  namespace: ibi-ns
+stringData:
+  .dockerconfigjson: <base64-docker-auth-code> <1>
+type: kubernetes.io/dockerconfigjson
+----
+<1> You must provide base64-encoded credential details. See the _Additional resources_ section for more information about using image pull secrets.
+
+.. Create the `Secret` resource for your image registry by running the following command:
++
+[source,terminal]
+----
+$ oc create -f secret-image-registry.yaml
+----
+
+. Optional. Configure static networking for the host:
+
+.. Create a `Secret` resource containing the static network configuration in `nmstate` format:
++
+.Example `host-network-config-secret.yaml` file
+[source,yaml]
+----
+apiVersion: v1 
+kind: Secret
+metadata:
+ name: host-network-config-secret <1>
+ namespace: openshift-machine-api
+type: Opaque
+stringData:
+ nmstate: | <2>
+  interfaces:
+    - name: ens1f0 <3>
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: true
+        address:
+          - ip: 10.6.85.8
+            prefix-length: 24
+        dhcp: false <4>
+        autoconf: false <5>
+      ipv6:
+        enabled: false
+  dns-resolver:
+    config:
+      server:
+        - 10.6.73.2 <6>
+        - 10.6.73.4
+  routes:
+    config: <7>
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 10.6.85.254
+        next-hop-interface: ens1f0
+        table-id: 254
+----
+<1> Specify the name for the `Secret` resource.
+<2> Define the static network configuration in `nmstate` format.
+<3> Specify the name of the interface on the host. The name of the interface must match the actual NIC name as shown in the operating system.
```

Comment:
@tsorya should we mention here that we support mac identifier? do we still support it after moving the network config to the BMH secret?

Reply:
Yep, why not? I mean it doesn't matter how you provide data